source/install/config-proxy-nginx.rst

```diff
@@ -71,20 +71,28 @@ NGINX is configured using a file in the ``/etc/nginx/sites-available`` directory
 
 6. Restart NGINX.
 
-  On Ubuntu 14.04 and RHEL 6.6:
-  
-  ``sudo service nginx restart``
-  
-  On Ubuntu 16.04 and RHEL 7.1:
-  
-  ``sudo systemctl restart nginx``
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service nginx restart``
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl restart nginx``
 
 7. Verify that you can see Mattermost through the proxy.
 
   ``curl http://localhost``
-  
+
   If everything is working, you will see the HTML for the Mattermost signup page.
 
-**What to do next**
+8. Make sure that Mattermost accepts connections only from the proxy.
```

Comment:
My understanding is that in HA, you can't modify the settings in the System Console because config.json is identical on each server and the System Console is read-only. The only way to update the config is to SSH into the machine and modify config.json directly.

The only time you'd need the System Console is when updating config changes without any downtime, in which case you have to use the System Console at each machine to reload the config. I added a paragraph that should cover this.

I changed {IP-address}  to {IP-or-domain} to make it more clear that ListenAddress can take a domain as well as an address.